### PR TITLE
Patch Grammatica to compile on Ubuntu 18.04 and 20.04

### LINKED ISSUE #379: 'make' fails on Ubuntu 18.04 and 20.04
I am trying to run specification tests for v1.1.0 on Ubuntu 18.04 and 20.04, but they seem to fail due to building problems of Grammatica:
```
+ cd grammatica-1.6
+ ant -k
Buildfile: /home/andrius/optimade/specification/tests/tools/grammatica-1.6/build.xml

codegen:

compile-clean:
    [mkdir] Created dir: /home/andrius/optimade/specification/tests/tools/grammatica-1.6/classes

compile-java:
    [javac] Compiling 96 source files to /home/andrius/optimade/specification/tests/tools/grammatica-1.6/classes
    [javac] warning: [options] bootstrap class path not set in conjunction with -source 1.4
    [javac] error: Source option 1.4 is no longer supported. Use 6 or later.
    [javac] error: Target option 1.5 is no longer supported. Use 1.6 or later.
Target 'compile-java' failed with message 'Compile failed; see the compiler error output for details.'.
Target 'codegen' failed with message 'The following error occurred while executing this line:
/home/andrius/optimade/specification/tests/tools/grammatica-1.6/build.xml:76: Compile failed; see the compiler error output for details.'.
```
[snip]
```
BUILD FAILED
/home/andrius/optimade/specification/tests/tools/grammatica-1.6/build.xml:36: The following error occurred while executing this line:
/home/andrius/optimade/specification/tests/tools/grammatica-1.6/build.xml:76: Compile failed; see the compiler error output for details.
```
After that, all tests fail due to missing JAR. Pinging @sauliusg and @rartino.

By the way, starting with Ubuntu 20.04, Grammatica appears in standard Ubuntu repositories as `libgrammatica-java`.

### PR DESCRIPTION
**Pull request by merkys**
Fixes #379. This PR implements my solution to set source and target to 1.6 for `javac` in Grammatica's `build.xml`.

### CODE REVIEW

File: dependencies/Debian-8.6/install.sh [hunk 1]
```diff
@@ -3,4 +3,5 @@
 sudo apt-get install -y \
     default-jdk \
     unzip \
+    xmlstarlet \
```
Comment (ml-evs) [inline dependencies/Debian-8.6/install.sh:6]: Should this also be added to the `Dockerfile`? Or even, should this script be unified with the installation step on the Dockerfile?

Comment (merkys) [reply dependencies/Debian-8.6/install.sh:6]: `Dockerfile` (as used by the CI) seems to fetch a bit newer version of Java, and that one does not complain about source/target versions for Grammatica. Thus the CI is immune to #379. I have not tried using `Dockerfile` locally, though.

**Review by rartino — APPROVED** [review]
Looks good!

**Comment by merkys** [comment]
FYI, Grammatica is included in standard Ubuntu repositories as `libgrammatica-java`, starting from 20.04. We could considerably shorten the CI runs by depending on `libgrammatica-java` and using prebuilt JAR. However, we should still retain current Makefile rules to fetch and build Grammatica for older/other OSes.

**Comment by ml-evs** [comment]
Superseded by #404

**Pull request closed without merging**